test/unit/models/users.test.js

```diff
@@ -87,5 +87,52 @@ describe("users", function () {
       expect(user.last_name).to.equal(userData.last_name);
       expect(userExists).to.equal(true);
     });
+
+    it("should add the github_user_id to the user collection", async function () {
+      const userData = userDataArray[0];
+      userData.github_user_id = "12345678";
+
+      const { isNewUser, userId } = await users.addOrUpdate(userData);
+
+      const data = (await userModel.doc(userId).get()).data();
+
+      expect(data.github_user_id).to.equal(userData.github_user_id);
+      expect(isNewUser).to.equal(true);
+    });
+
+    it("should update the github_id in the user collection", async function () {
+      const userData = userDataArray[0];
+      userData.github_id = "Yash Sinha";
+
+      // Add the user the first time
+      const { userId } = await users.addOrUpdate(userData);
+
+      // Update the user with same data and new github_user_id
+      userData.github_id = "Ankush Dharkar";
+      await users.addOrUpdate(userData, userId);
+
+      const data = (await userModel.doc(userId).get()).data();
+
+      expect(data.github_id).to.equal(userData.github_id);
+    });
+
+    it("github_user_id field should have a maximum length of 8 characters", async function () {
+      const userData = { ...userDataArray[0], github_user_id: "a".repeat(9) };
+
+      try {
+        await users.addOrUpdate(userData);
+      } catch (error) {
+        expect(error.message).to.equal("Validation error: github_id exceeds maximum length of 8 characters");
+      }
+    });
```

Comment:
This unit test is also pointless. Since the control never reaches the catch block and the expect statement is never executed. So this test will pass everytime by default

Reply:
I'll remove the unwanted tests.